packages/my-extended-retail-app/package.json

```diff
@@ -6,59 +6,59 @@
         "node": "^16.0.0 || ^18.0.0",
         "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
     },
-    "mobify": {
+    "ccExtensibility": {
         "extends": "retail-react-app",
         "overridesDir": "/pwa-kit-overrides"
     },
```

Comment:
Q for `overridesDir`:

It appears this is a configurable value, I'm trying to understand the use case of the configurability, when would you want to name this folder differently, than let's say a default `overrides` folder?

Reply:
a project might have other requirements for structure dictated by other `npm` packages or any number or various external tools we can't know about. We make `overridesDir` dynamic to future proof the project against unanticipated outside side-effects that would cause us to need to rush to push a major version fix

Reply:
Hmmm, one thing i noticed is that we have a tendency to over-optimize code when we don't have a solid use case! I can give you many examples that we created the "options" that was never used by customers, and users need to pay the price for more complexity.. 

Unless we have a solid use case, i'd advocate for starting with something simple!

Reply:
I guess my feeling is that comparing the setting of two values or one, both seem to me to be quite simple

I want to point out again, if some other project does something similar, which we have no way of anticipating, we will get **massively** hosed. Let's say this happens 3 years from now, we will have to do a fire drill to release a new major version, but **also** maybe get 500 customers to update. Even though it's a long shot, the "cost" is just one extra key / value pair. Seems like low cost to avoid high risk.

I agree with you that we shouldn't prematurely optimize, but when we bring in cost / risk analysis I think it's worth keeping this

Reply:
😛 @bfeister i'm not convinced that this will be a massively hosed issue, feels like an small additive non-breaking change to me.

```diff
- const overrideDir =  DEFAULT_OVERRIDE_FOLDER
+ const overrideDir = pkg.overrideDir + DEFAULT_OVERRIDE_FOLDER
```